<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2KN8RMI" name="Default test" tests="17" failures="1" timestamp="24 Jul 2018 10:55:42 GMT" time="0.163" errors="0">
  <testcase name="aD1test" time="0.006" classname="com.scp.TestTestng.DTest"/>
  <testcase name="aZryaDtest" time="0.002" classname="com.scp.TestTestng.DTest"/>
  <testcase name="abcDtest" time="0.002" classname="com.scp.TestTestng.DTest"/>
  <testcase name="apqerDtest" time="0.008" classname="com.scp.TestTestng.DTest"/>
  <testcase name="azeyrDtest" time="0.015" classname="com.scp.TestTestng.DTest"/>
  <testcase name="aZryactest" time="0.003" classname="com.scp.TestTestng.CTest"/>
  <testcase name="azeyrctest" time="0.002" classname="com.scp.TestTestng.CTest"/>
  <testcase name="aZryatest" time="0.002" classname="com.scp.maven.TestNg.ATest"/>
  <testcase name="abctest" time="0.003" classname="com.scp.maven.TestNg.ATest"/>
  <testcase name="apqertest" time="0.005" classname="com.scp.maven.TestNg.ATest"/>
  <testcase name="azeyrtest" time="0.005" classname="com.scp.maven.TestNg.ATest"/>
  <testcase name="m2" time="0.005" classname="com.scp.maven.TestNg.ATest">
    <failure type="java.lang.AssertionError" message="This test case is failed...!">
      <![CDATA[java.lang.AssertionError: This test case is failed...!
	at com.scp.maven.TestNg.ATest.m2(ATest.java:29)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- m2 -->
  <testcase name="axyzDtest" time="0.003" classname="com.scp.TestTestng.DTest"/>
  <testcase name="axyzctest" time="0.001" classname="com.scp.TestTestng.CTest"/>
  <testcase name="axyztest" time="0.001" classname="com.scp.maven.TestNg.ATest"/>
  <testcase name="abcctest" time="0.002" classname="com.scp.TestTestng.CTest"/>
  <testcase name="m1" time="0.0" classname="com.scp.maven.TestNg.ATest">
    <skipped/>
  </testcase> <!-- m1 -->
</testsuite> <!-- Default test -->
